### PR TITLE
[bugfix] lift without refresh

### PR DESCRIPTION
**Pull request by christojeffrey**
I'm trying to make the changes as minimal as possible, keeping the logic as much as I can

but you can actually make the code significantly more efficient

### CODE REVIEW

**Comment by christojeffrey** [comment]
aand this also remove the need for preGraphData

File: svelte/src/routes/v2/scripts/filter/lift-edges.ts [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+	GraphData,
+	GraphDataNode
 } from '../../types';
 
 export interface FilteredNode extends ConvertedNode {
 	parent?: FilteredNode;
 }
-
+// HELPER FUNCTIONS
 // Find length of common prefix of 2 string arrays
```
Comment (Adityapnn811) [inline svelte/src/routes/v2/scripts/filter/lift-edges.ts]: I think the name of the function and description should be changed before it turns into confusion

Comment (christojeffrey) [reply svelte/src/routes/v2/scripts/filter/lift-edges.ts]: will do, wait

Comment (christojeffrey) [reply svelte/src/routes/v2/scripts/filter/lift-edges.ts]: done

**Comment by christojeffrey** [comment]
this handles task below

image

**Review by Perry26 — APPROVED** [review]

**Pull request merged**